metagraph/integration_tests/test_align.py

```diff
@@ -52,7 +52,7 @@ def test_simple_align_all_graphs(self, representation):
         self.assertEqual('nodes (k): 16438', params_str[1])
         self.assertEqual('canonical mode: no', params_str[2])
 
-        stats_command = '{exe} align -i {graph} --align-vertical-bandwidth 1000000 --discovery-fraction 0.0 {reads}'.format(
+        stats_command = '{exe} align -i {graph} --align-vertical-bandwidth 1000000 --align-min-exact-match-threshold 0.0 {reads}'.format(
```

Comment:
"min" and "threshold" are synonyms here. Use just one of them.

--align-min-exact-match
or
--align-exact-match-threshold

Reply:
I used `align-min-exact-match`